internal/infrastructure/repositories/qoveryapi/secret_qoveryapi_models.go

```diff
@@ -27,11 +27,17 @@ func newDomainSecretFromQovery(v *qovery.Secret) (*secret.Secret, error) {
 		return nil, secret.ErrNilSecret
 	}
 
+	description := ""
+	// shouldnt we have a description on qovery.Secret?
```

Comment:
We should, it must be added in our openapi repo (there are 2 different objects for Secret vs Variable)

Reply:
https://github.com/Qovery/qovery-openapi-spec/blob/main/openapi.yaml#L16979